dts/common/nordic/nrf54l15.dtsi

```diff
@@ -577,6 +578,22 @@
 				interrupts = <270 NRF_DEFAULT_IRQ_PRIORITY>;
 				status = "disabled";
 			};
+
+			regulators: regulator@120000 {
+				compatible = "nordic,nrf54l-regulators";
+				reg = <0x120000 0x1000>;
+				status = "okay";
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				vregmain: regulator@120600 {
+					compatible = "nordic,nrf5x-regulator";
+					reg = <0x120600 0x1>;
+					status = "okay";
```

Comment:
this should be enabled at board level, see other families

Reply:
is non-DCDC an option, right?

Reply:
yes, it is also an option

Reply:
I moved enabling and DCDC configuration to board level